circleci/docker-publish

```diff
@@ -55,13 +53,13 @@ fi
 
 # dockerhub login. 
 # `FROM image` in Dockerfile is going to pull from DockerHub
-docker login -u $DOCKER_USER -p $DOCKER_PASS --email="$DOCKER_EMAIL"
+docker login -u $DOCKER_USER -p $DOCKER_PASS
```

Comment:
How about just changing this one line?

What about:

```
docker login -u $DOCKER_USER -p $DOCKER_PASS --email="$DOCKER_EMAIL" || docker login -u $DOCKER_USER -p $DOCKER_PASS
```

should be reverse compatible